.env

```diff
@@ -3,4 +3,5 @@
 VITE_SETTINGS_API_URL=http://localhost:3000/api/settings/
 VITE_VIVAWALLET_TRANSACTION_ORDER=http://localhost:3000/api/vivawallet/transaction_order/
 VITE_VIVAWALLET_TRANSACTION_VERIFICATION=http://localhost:3000/api/vivawallet/transaction_verification/
-VITE_STRIPE_PUBLIC_KEY=pk_test_51NTszUJEpEVZWKtc2yNFPJ9DYSmhNnf04vCUmbM3fMS94WK2w1YuhiTcxMIti8p3etufbrsr1oJpG2OUaLUmNUTU00cxAmOXLZ
\ No newline at end of file
+VITE_STRIPE_PUBLIC_KEY=pk_test_51NTszUJEpEVZWKtc2yNFPJ9DYSmhNnf04vCUmbM3fMS94WK2w1YuhiTcxMIti8p3etufbrsr1oJpG2OUaLUmNUTU00cxAmOXLZ
+VITE_AUGUSTIN_COVER=https://augustin.or.at/wp-content/uploads/2023/08/Augustin_580_Cover_web.jpg
```

Comment:
You added "VITE_AUGUSTIN_COVER" as a new variable but it is not used anywhere...